calicoctl/resourcemgr/networkpolicy.go

```diff
@@ -29,7 +29,7 @@ func init() {
 		true,
 		[]string{"networkpolicy", "networkpolicies", "policy", "np", "policies", "pol", "pols"},
 		[]string{"NAME"},
-		[]string{"NAME", "ORDER", "SELECTOR", "NAMESPACE"},
```

Comment:
Is there a good reason for removing the namespace from `-o wide` besides that's what k8s does? I find having namespace field in `-o wide` very useful

Reply:
If we keep the `NAMESPACE` field in `-o wide` output, this could be super simple (just move `NAMESPACE` first on this list, and we're done (since calicoctl already prints the correct namespace, just not in the first column). And, I think having the namespace in `-o wide` would be super useful, we can also add the namespace field for the normal `ps` output.

Reply:
Yeah the only reason is to copy how K8s does it. As you can see in the output you basically only get no namespace when you aren't specifying the namespace or requesting all-namespaces. My initial thought was just to move Namespace to the first column, @caseydavenport requested it behave like kubectl.